From 71e762fb29a8ba218c90a48e0e160b2bb8b74326 Mon Sep 17 00:00:00 2001
From: Alexandre Cassen <acassen@gmail.com>
Date: Mon, 6 Jan 2014 15:04:28 +0100
Subject: [PATCH 1/7] ipvs: make nlerr2syserr libnl dependent

nlerr2syserr() is only used when libnl is present... simply reflect
this in libipvs.
---
 keepalived/libipvs-2.6/libipvs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/keepalived/libipvs-2.6/libipvs.c b/keepalived/libipvs-2.6/libipvs.c
index 223e76b..8ccf94d 100644
--- a/keepalived/libipvs-2.6/libipvs.c
+++ b/keepalived/libipvs-2.6/libipvs.c
@@ -54,6 +54,7 @@ static int family, try_nl = 1;
 	CHECK_IPV4(s, ret);					\
 	CHECK_PE(s, ret);
 
+#ifdef LIBIPVS_USE_NL
 #ifndef FALLBACK_LIBNL1
 static int nlerr2syserr(int err)
 {
@@ -79,7 +80,6 @@ static int nlerr2syserr(int err)
 }
 #endif
 
-#ifdef LIBIPVS_USE_NL
 struct nl_msg *ipvs_nl_message(int cmd, int flags)
 {
 	struct nl_msg *msg;
-- 
1.8.1.4

